### PR TITLE
SSCS-9399 CCD E2E tests - Add UI tests for "provideAppointeeDetails" event

### PR DESCRIPTION
**Pull request by bensonlievoco**
Add UI tests for "provideAppointeeDetails" event

### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/SSCS-9399

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by rasikasri — APPROVED** [review]

File: e2e/features/provide-appointee-details.feature [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+Feature: The lapse
+
+  Background:
+    Given I presetup an "PIP" SYA case
+    And I am signed in as a Case Officer
+    And I navigate to an existing case
+
+  @provide-appointee-details
+  Scenario: Should end up in "With DWP" state when ALL fields are present
+    When I choose "Lapse appeal"
```
Comment (gokul-sol) [inline e2e/features/provide-appointee-details.feature:11]: @bensonlievoco Not sure if we need the lapse appeal steps as they mention a different functionality.
All we need to do is create a new case and add appointee details to the case and verify if they are added against the case

Comment (bensonlievoco) [reply e2e/features/provide-appointee-details.feature]: Just updated

File: e2e/features/provide-appointee-details.feature [hunk 2]
```diff
@@ -0,0 +1,21 @@
+@migrated-to-exui @nightly-test
+Feature: The lapse
```
Comment (divisathyan) [inline e2e/features/provide-appointee-details.feature:2]: Is this the correct feature name?

**Review by divisathyan — COMMENTED** [review]
One small comment

**Pull request merged**